src/App/Fossa/ManualDeps.hs

```diff
@@ -88,13 +89,17 @@ toSourceUnit root manualDeps@ManualDependencies{..} maybeApiOpts = do
       referenceLocators = refToLocator <$> referencedDependencies
       additional = toAdditionalData (NE.nonEmpty customDependencies) (NE.nonEmpty remoteDependencies)
       build = toBuildData <$> NE.nonEmpty (referenceLocators <> archiveLocators)
+      originPath = case depsFile of
+        (ManualJSON path) -> toRelativePath root path
+        (ManualYaml path) -> toRelativePath root path
```

Comment:
is there a way to do something similar to `(_ path)`? I couldn't figure this one out

Reply:
No, this is actually intentional, since the different constructors might have different arities (arity: number of arguments needed).  There's likely a proposal/extension for allowing something like this, but I don't know about it.

Reply:
good to know, thanks